src/scales/scale.time.js

```diff
@@ -175,8 +175,23 @@ module.exports = function(Chart) {
 		var ticks = [];
 		if (options.maxTicks) {
 			var stepSize = options.stepSize;
-			ticks.push(options.min !== undefined ? options.min : niceRange.min);
-			var cur = moment(niceRange.min);
+			var startTick = options.min !== undefined ? options.min : niceRange.min;
+			var seniorUnitStart = moment(startTick).add(1, options.seniorUnit).startOf(options.seniorUnit);
```

Comment:
We discussed in the feature request https://github.com/chartjs/Chart.js/issues/4187 that the alignment always be on. Can you auto-detect what the majorUnit is? You should be able to use `determineUnit` and `interval` to do so. I think this would be much simpler for the user